opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/FileManager.java

```diff
@@ -588,10 +585,16 @@ public OpenCGAResult<File> create(String studyStr, FileCreateParams createParams
             ParamUtils.checkParameter(createParams.getPath(), "path");
             ParamUtils.checkObj(createParams.getType(), "type");
 
+            boolean isResource = createParams.getResource() != null && createParams.getResource();
```

Comment:
works the same and slightly shorter if you do `createParams.getResource() == Boolean.TRUE`